apps/frontend/src/libs/components/button/styles.module.css

```diff
@@ -13,6 +13,28 @@
 	transition: background-color 0.3s ease;
 }
 
-.button:hover {
+.button:hover:not(:disabled) {
 	background-color: var(--color-brand-primary-hover);
 }
+
+.button-danger {
+	background-color: var(--color-danger);
+}
+
+.button-danger:hover:not(:disabled) {
+	background-color: var(--color-danger-hover);
+}
+
+.button-outlined {
+	background-color: transparent;
+	border: 1px solid var(--color-brand-primary);
+}
+
+.button-outlined:hover:not(:disabled) {
+	background-color: var(--color-background-hover);
+}
+
+:disabled {
+	cursor: default;
+	opacity: 0.5;
+}
```

Comment:
not sure if this selector matches anything 🤔 also let's use not allowed cursor instead of default

Reply:
Oops, missed `.button` (however that works somehow)